Voycar.Api.Web/Generic/Endpoint/Delete/Single.cs

```diff
@@ -7,18 +7,18 @@ public abstract class Single<TEntity>
     where TEntity : Entity
 {
     protected readonly IRepository<TEntity> _repository;
-    protected readonly string[] roles;
+    protected new readonly string[] Roles;
```

Comment:
Was macht das new da? Ist das nicht überflüssig?
```suggestion
    protected readonly string[] Roles;
```